repo-template/.vscode/extensions.json

```diff
@@ -0,0 +1,10 @@
+{
+    "recommendations": [
+        "sumneko.lua",
+        "overextended.cfxlua-vscode",
+        "ihyajb.qbcore-code-snippets",
```

Comment:
Do we still need these snippets? lol

Reply:
I think they're still helpful sometimes 🤔